src/pages/Tests.tsx

```diff
@@ -73,12 +74,22 @@ const DROP_FILL_GAPS_SOLUTION = ['on', 'at', 'under', 'in', 'through'];
 const Tests = () => {
   return (
     <>
-      <BoxDrop options={BOX_DROP_OPTIONS} solution={BOX_DROP_SOLUTION} />;
-      <DropFillGaps
-        sentences={DROP_FILL_GAPS_SENTENCES}
-        options={DROP_FILL_GAPS_OPTIONS}
-        solution={DROP_FILL_GAPS_SOLUTION}
-      />
+      <Box mb={15}>
+        <Typography textAlign={'center'} variant='h4' mb={2}>
```

Comment:
Typography has a built-in prop that just align="center" , maybe you can change the textAlign properties.